src/pages/map.adoc

```diff
@@ -9,8 +9,7 @@ public transportation between the main destinations that will be
 relevant for you during your trip. Please bear in mind that although
 biking is the favorite method of transportation, this is not covered
 here. If you want to know how to get to the office by bike, you can use
-https://www.google.com/maps[Google Maps] and enter the following
-address: *Van Marwijk Kooystraat 15, 1114 AG, Amsterdam*
+Google Maps {url-picnic-hq-gmaps}.
```

Comment:
Will this render the link under `Google Maps`?  :thinking:

Reply:
Nope, will render like this 
![image](https://user-images.githubusercontent.com/3147756/72681257-075fb280-3ac2-11ea-88eb-c82f90b47d6c.png)

Reply:
Hmm, that seems to break the sentence. But I guess it won't confuse anybody.

Reply:
I agree that this is not ideal, but I feel that it is fairly hard to weave this token in a sentence meaningfully.